dashboard/app/views/levels/editors/fields/_authored_hints.haml

```diff
@@ -70,10 +70,10 @@
                 const xmlContainer = space.find('.markdown_preview').get(0);
 
                 // Prevent rendering multiple copies of a block svgs by removing
-                // all Blockly SVG containers before converting any XML into blocks.
-                const blocklySvgs = xmlContainer.querySelectorAll('svg.blocklySvg.readOnlyBlockSpace');
+                // all SVG containers before converting any XML into blocks.
+                const svgs = xmlContainer.querySelectorAll('svg');
```

Comment:
I haven't tested this, but I think we could use `querySelectorAll('svg[class*="readOnlyBlockSpace"]');` to most accurately select workspaces made with either version of Blockly, and no other svgs.

https://stackoverflow.com/questions/8714090/how-to-do-a-wildcard-element-name-match-with-queryselector-or-queryselector

Reply:
I'll try it out - thanks!

Reply:
It work - thanks!